### DISCUSSION TITLE
Spread constraints with the cluster operator HA setup.

### DISCUSSION THREAD

**Discussion by OuesFa** [strimzi/strimzi-kafka-operator#7400] [General]
Hello 👋 

AFAIU, the cluster operator leader election allows to have standby replicas which are useful when a node crashes and Kube is unable to reschedule the operator pod into another node, so it doesn't make sense to create the standby pod within the same node as the leader pod. 

Are there any default spread constraints or anti affinity rules ensuring that standby instances are scheduled in a different node / zone if possible? 

I tried to take a look in the PR but could not find anything like that https://github.com/strimzi/strimzi-kafka-operator/pull/7177/files

If such rules / constraint do not exist, does it make sense to create them? Would it be a relatively simple contribution? 

Thank you very much for your excellent work!
❤️ Strimzi

**Comment by scholzj** [comment]
Having a default spread which would work for everyone is not easy since everyone has different requirements, wants to use different topology for the spread etc. So yes, it makes sense to use the spread of podAntiAffinity. But this might be unique for everyone. So the best way is to just go and add your own configuration to match your environment.

**Reply by OuesFa** [reply]
Makes sense 👍
